scanpipe/pipelines/populate_purldb.py

```diff
@@ -36,24 +40,60 @@ def steps(cls):
 
     def populate_purldb_with_discovered_packages(self):
         """Add DiscoveredPackage to PurlDB."""
+        packages = self.project.discoveredpackages.all()
+        package_urls = [pacakage.purl for pacakage in packages]
         self.feed_purldb(
-            packages=self.project.discoveredpackages.all(),
+            package_urls=package_urls,
             package_type="DiscoveredPackage",
         )
 
     def populate_purldb_with_discovered_dependencies(self):
```

Comment:
We want the least amount of code logic inside the pipeline. It'll be better to move the following as a reusable pipe ;)